.circleci/config.yml

```diff
@@ -67,7 +67,10 @@ jobs:
               fi
       - run:
           name: Deploy
-          command: ./deploy.sh
+          command: |
+              if [ ! $CIRCLE_BRANCH = 'master' ]; then
```

Comment:
This is incorrect.
Deploygate upload task should be run when the circle branch is `master` branch.

Reply:
I'm sorry to trouble you. 😭 
I succeed to build, thanks to you!